src/BenchmarksServer/Startup.cs

```diff
@@ -214,13 +224,22 @@ private static string CloneRestoreAndBuild(string path, ServerJob job)
 
             AddSourceDependencies(path, benchmarksDir, dirs);
 
-            var benchmarksPath = Path.Combine(path, benchmarksDir, "src", "Benchmarks");
+            // Install latest SDK and runtime
+            // * Use custom install dir to avoid changing the default install,  which is impossible if other processes
+            //   are already using it.  Custom install is located under default install, so deleting the default
+            //   install will delete both.
```

Comment:
The goal was to re-use the custom dotnet folder across runs to save time.  Using a temp dir will make the first run after starting the BenchmarksServer slow, but I agree this is a better solution since the BenchmarksServer is a long-running process.